k8s/celery.yaml.j2

```diff
@@ -37,4 +37,24 @@ spec:
             limits:
               cpu: {{ CELERY_WORKER_CPU_LIMIT }}
               memory: {{ CELERY_WORKER_MEMORY_LIMIT }}
+        - name: {{ CELERY_SCHEDULER_NAME }}
```

Comment:
@limed All this is a duplicate-and-tweak of the main `worker`, above in this file.

Can you please engage laser-eyes to check it makes sense (comparing with the change to `docker-compose.yml` will help) and if you're happy add the env vars, please (ticketed as #413)?

Reply:
This does look fine but what needs to happen as well is that we should set sane values in the `Makefile`

Issue #413 exposes it as a k8s env var but I think we should set this in the `Makefile`

See: https://github.com/mdn/developer-portal/blob/master/k8s/Makefile#L34-L40 we can set sane values here and then if need be override them under the `k8s/config/*.sh` files, does that make sense?

Reply:
Thanks. Will remove this code and we'll def go with your from PR #422